code-challenges.js

```diff
@@ -7,47 +7,152 @@
+// Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: 
+//"42 is below boiling point",
+ //"350 is above boiling point", 
+ //"212 is at boiling point"
 
 // Pseudo code:
-
+//create a function that takes in a given number named it boilingPoint
+// if statement to check for anything 42 and below, if true return ...
+//else if the temp = 212 say its at boiling
+//else if temp is above 350 return is above boiling point
+// stretch can I do this with another shorter one line method? Would it be inerpolation?
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const boilngPoint = (num) => {
+    if (num < temp1 ){
+        return "temp is below boiling point"; 
+    }else if
+        (num = temp3){
+            return "is at boiling point";
+        } else if (num > temp2){ 
+            return "is above boiling point";
+        }
+    };
+
+console.log(boilngPoint(212))
+// condition ? output if true : output if false
+const boilBchecker 
+
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 // Pseudo code:
 
-const myNumbers1 = [3, 7, 0, 36, -9]
-const myNumbers2 = [8, -7, 42, 9, 13]
+const myNumbers1 = [3, 7, 0, 36, -9]  //1st array
+const myNumbers2 = [8, -7, 42, 9, 13]  //2nd array
 
+const lengthOf = myNumbers1.concat(myNumbers2);    //declaring a function and saying to concat them together into one array //
+console.log (lengthOf.length); //return the new joined array's length expected output 10
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 ohcE"
 
 // Pseudo code:
-
-const currentCohort = "Echo 2022"
-
+//in a console log us .split turn string into array then .join takes an array and returns a string then reverse it  /with .reverse //
+const currentCohort = "Echo 2022"  
+console.log(currentCohort.split(" ").join().reverse(" "));
+
+//Psuedo code:
+//another way to solve: with a decrimenting loop in a function so code can be resuable
+ //declare a function backwards that can accept an argument of a string
+//make a variable to hold the new string//
+//for loop to iterate thru the string and starting at index -1 the last letter of the string,while i is >= o the loop //will iterate
+//adding the data at each index into a new string
+// return the string reversed
+//here we use the function created above to reverse the currentCohort//
+const currentCohort = "Echo 2022"          
+function backwards(str) { 
+    var newCohort = "";               
+    for (var i = str.length - 1; i>= 0; i-- ) {    
+        newCohort += str[i]; 
+    }
+    return newCohort; 
+}
+const reverseLean = backwards(currentCohort); 
+console.log(reverseLean); 
 
```

Comment:
great work. Showing various ways to solve all your coding challenges is what will push you to the next level.